src/Altinn.Platform/Altinn.Platform.Authentication/Authentication/Dockerfile

```diff
@@ -1,12 +1,12 @@
-FROM mcr.microsoft.com/dotnet/core/sdk:3.1.405-alpine3.12 AS build
+FROM mcr.microsoft.com/dotnet/sdk:5.0.102-1-alpine3.12 AS build
```

Comment:
@TheTechArch Latest is `5.0.103-alpine3.13` and there are [security fixes](https://github.com/dotnet/core/blob/master/release-notes/5.0/5.0.3/5.0.3.md#notable-changes).

Reply:
Let's do that as a separate PR.

Reply:
I don't see why that is better, but as long as this is done before next deploy it's fine by me.